internal/cli/options.go

```diff
@@ -13,13 +13,14 @@ import (
 )
 
 type Options struct {
-	Migrate        bool
-	Verbose        bool
-	InitRegistry   bool
-	NonInteractive bool
-	OutputFormat   string
-	logger         *zap.SugaredLogger
-	Registry       *persistency.Registry //will be initialized during CLI bootstrap in main.go
+	Migrate           bool
+	Verbose           bool
+	InitRegistry      bool
+	NonInteractive    bool
+	OutputFormat      string
+	logger            *zap.SugaredLogger
+	Registry          *persistency.Registry //will be initialized during CLI bootstrap in main.go
+	OccupancyTracking bool
```

Comment:
As mentioned above, we have to align with the team how we deal in the long-term with feature-flags.